src/ddsolver/dds.py

```diff
@@ -17,7 +17,11 @@
 from ctypes import *
 from sys import platform
 
-dds = cdll.LoadLibrary("./ddsolver/dds.dll") if platform == "win32" else cdll.LoadLibrary("./ddsolver/libdds.so")
+from pathlib import Path
```

Comment:
i have also changed this dds loading a bit.
moved the library files to `ben/bin` and also added a `BEN_HOME` environment variable but kept it optional.
please take a look at the latest main branch, there might be merge conflicts